.gitignore

```diff
@@ -7,7 +7,8 @@ cypress/videos/
 test/example-recordings.json
 .idea/
 cypress.env.json
-.env*
```

Comment:
I think we still want to exclude all `.env*` files with `.env.sample` being the exception so we (read: me because i `git add .` freely) don't accidentally add `.env.local` to git.